src/openapi/openapi.json

```diff
@@ -3815,6 +3944,242 @@
         "x-router-controller": "cors"
       }
     },
+    "/api/data/submission/operation/createStepReviewApproval": {
```

Comment:
A copy of this endpoint is needed with the path of `/api/submission/createStepReviewApproval` with [here](Please add a comment to explain how this if statement works as well as parentheses to make it more clear.) replaced with `"Service_Authorizer": []`